database/seeders/CitySeeder.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\City;
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class CitySeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        City::factory(25)->create();
```

Comment:
Use CityFactory directly instead of City::factory to avoid giving more responsibilities to the model
Doc: https://lightit.slite.com/app/docs/wK2Zeab_ha-etL/No-darle-la-responsabilidad-al-modelo-de-crear-sus-Factories